src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/TableLayoutPanelResizeGlyph.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Drawing;
+
+namespace System.Windows.Forms.Design.Behavior;
+
+internal class TableLayoutPanelResizeGlyph : Glyph
+{
+    private Rectangle bounds;
+    private Cursor hitTestCursor;
+    private TableLayoutStyle style;
+    private TableLayoutResizeType type;
+
+    /// <summary>
+    /// This constructor caches our necessary state and determine what 'type' it is.
+    /// </summary>
+    internal TableLayoutPanelResizeGlyph(Rectangle controlBounds, TableLayoutStyle style, Cursor hitTestCursor, Behavior behavior) : base(behavior)
+    {
+        bounds = controlBounds;
+        this.hitTestCursor = hitTestCursor;
+        this.style = style;
+
+        if (style is ColumnStyle)
+        {
+            type = TableLayoutResizeType.Column;
+        }
+        else
+        {
+            type = TableLayoutResizeType.Row;
+        }
+    }
+
+    /// <summary>
+    /// Represents the bounds of the row or column line being rendered by the TableLayoutPanelDesigner.
+    /// </summary>
+    public override Rectangle Bounds
+    {
+        get
+        {
+            return bounds;
+        }
+    }
+
+    /// <summary>
+    /// Represents the Style associated with this glyph: Row or Column.
```

Comment:
FYI - we are indenting content of `///` comments by 2 spaces. Please apply to all files in this PR.
```suggestion
    ///  Represents the Style associated with this glyph: Row or Column.
```

Reply:
Done.